### PR TITLE
my initial homework commit

### PR DESCRIPTION
**Pull request by mihailfox**
I've added the code for lesson 1 homework

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Program.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4828805&bid=16044864)

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,32 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+			int numberOfPersons = 2;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Add the 'const' modifier to 'numberOfPersons'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4828805)

Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4828805)

**Comment by Almantask** [comment]
You reverted your changes, are you sure you did i the right thing?

**Comment by Alumniminium** [comment]
@mihailfox created another pull request, so I'm closing this one.

**Pull request closed without merging**